build.gradle.kts

```diff
@@ -23,7 +23,7 @@ repositories {
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(21))
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
Why? Is the integrator using java17? Do we need to amend also the GH action configuration?

Reply:
The issue is that the integrator wants to use the library directly in Android which is limited to JDK 17

Reply:
I am seeing if targeting JDK 17 is enough, or whether the workflow also needs downgrading